docs/data/material/guides/theme-scoping/theme-scoping.md

```diff
@@ -0,0 +1,95 @@
+# Theme scoping
+
+<p class="description">Learn how to use multiple styling solutions in a single Material UI app.</p>
+
+:::warning
+Having more than one styling library can introduce unnecessary complexity to your project.
+You should have a very good reason to do this.
+:::
+
+Starting with [v5.12.0](https://github.com/mui/material-ui/releases/tag/v5.12.0), Material UI can coexist with other component libraries that depend on Emotion or styled-components—this is possible thanks to _theme scoping_.
+
+To do this, you need to render Material UI's `ThemeProvider` as an inner provider and use the `THEME_ID` to store the theme, as shown below:
+
+```js
+import { ThemeProvider, THEME_ID, createTheme } from '@mui/material/styles';
+import { AnotherThemeProvider } from 'another-ui-library';
+
+const materialTheme = createTheme(…your theme);
```

Comment:
this would crash if you copy and paste it, it also looks strange:

<img width="616" alt="Screenshot 2023-08-14 at 02 19 21" src="https://github.com/mui/material-ui/assets/3165635/4a8a8cb6-e692-4a7e-a7bb-b960f4c4bf21">


So maybe:

```suggestion
const materialTheme = createTheme(/* your theme */);
```

<img width="621" alt="Screenshot 2023-08-14 at 02 19 17" src="https://github.com/mui/material-ui/assets/3165635/b837b6e6-a5bb-4815-b846-1f4c0d6847e4">